src/main/java/edu/harvard/iq/dataverse/dashboard/DashboardUsersPage.java

```diff
@@ -0,0 +1,199 @@
+
+        this.userList = userListResult.getUserList();
+        this.pager = userListResult.getPager();
+        
+        return true;
+        
+    }
+
+
+    
+    public String getListUsersAPIPath() {
+        //return "ok";
+        return Admin.listUsersFullAPIPath;
+    }
+
+    /** 
+     * Number of total users
+     * @return 
+     */
+    public String getUserCount() {
+        //return userService.getTotalUserCount();
+
+        return NumberFormat.getNumberInstance(Locale.US).format(userService.getTotalUserCount());
+    }
+
+    /** 
+     * Number of total Superusers
+     * @return 
+     */
+    public Long getSuperUserCount() {
+        
+        return userService.getSuperUserCount();
+    }
+
+    public List<AuthenticatedUser> getUserList() {
+        return this.userList;
+    }
+
+    /**
+     * Pager for when user list exceeds the number of display rows
+     * (default: UserListMaker.ITEMS_PER_PAGE)
+     * 
+     * @return 
+     */
+    public Pager getPager() {
+        return this.pager;
+    }
+
+    public void setSelectedPage(Integer pgNum){
+        if ((pgNum == null)||(pgNum < 1)){
+            this.selectedPage = 1;
+        }
+        selectedPage = pgNum;
+    }
+
+    public Integer getSelectedPage(){
+        if ((selectedPage == null)||(selectedPage < 1)){
+            setSelectedPage(null);            
+        }
+        return selectedPage;
+    }
+    
+    public String getSearchTerm() {
+        return searchTerm;
+    }
+
+    public void setSearchTerm(String searchTerm) {
+        this.searchTerm = searchTerm;
+    }
+    
+    /* 
+       Methods for toggling the supeuser status of a selected user. 
+       Our normal two step approach is used: first showing the "are you sure?" 
+       popup, then finalizing the toggled value. 
+    */
+       
+    AuthenticatedUser selectedUser = null; 
+    
+    public void setSelectedUser(AuthenticatedUser user) {
+        this.selectedUser = user;
+    }
+    
+    public AuthenticatedUser getSelectedUser() {
+        return this.selectedUser;
+    }
+    
+    
+    public void setUserToToggleSuperuserStatus(AuthenticatedUser user) {
+        logger.info("selecting user "+user.getIdentifier());
+        selectedUser = user; 
+    }
+    public void saveSuperuserStatus(){
+        logger.info("Toggling user's "+selectedUser.getIdentifier()+" superuser status; (current status: "+selectedUser.isSuperuser()+")");
+        logger.info("Attempting to save user "+selectedUser.getIdentifier());
+        authenticationService.update(selectedUser);
+        
+    }
+    
+    public void cancelSuperuserStatusChange(){
+        logger.info("unToggling user's "+selectedUser.getIdentifier()+" superuser status; (current status: "+selectedUser.isSuperuser()+")");
+        selectedUser.setSuperuser(!selectedUser.isSuperuser());
+        
+    }
+    
+    private void msg(String s){
+        System.out.println(s);
```

Comment:
Can we please use our `logger.fine` convention instead of these `msg` functions? That way we can control the logging level at runtime: http://guides.dataverse.org/en/4.6.2/developers/debugging.html